src/server/pkg/storage/chunk/writer.go

```diff
@@ -18,299 +18,249 @@ import (
 const (
 	// WindowSize is the size of the rolling hash window.
 	WindowSize = 64
-	// (bryce) this should be configurable.
-	bufSize = 100 * units.MB
 )
 
 // initialWindow is the set of bytes used to initialize the window
 // of the rolling hash function.
 var initialWindow = make([]byte, WindowSize)
 
+// Annotation is used to associate information with data
+// written into the chunk storage layer.
+type Annotation struct {
+	RefDataRefs []*DataRef
+	NextDataRef *DataRef
+	tags        []*Tag
```

Comment:
Is `tags` supposed to be private? It seems like you refactored Annotation to be a pure model (which I agree with).  I would: 1. export tags 2.move it to a separate structure for associated state or 3. move it to the bottom and separate by a new line

Reply:
It is supposed to be unexported. It is a field used internally by the chunk writer. I can move it to the bottom and separate it by a new line for now, but I think we will revisit this model post alpha.